graphdatascience/query_runner/gds_arrow_client.py

```diff
@@ -30,14 +32,20 @@ def create(
         tls_root_certs: Optional[bytes] = None,
         connection_string_override: Optional[str] = None,
     ) -> "GdsArrowClient":
-        arrow_info = query_runner.call_procedure(endpoint="gds.debug.arrow", custom_error=False).squeeze().to_dict()
+        arrow_info = (
```

Comment:
Is there a minimum supported GDS version policy for the client? Since `advertisedListendAddress` has been renamed in 2.3, do we need a check for server versions before that, or is it ok to fail on those?

Reply:
I thought listen address was there from the beginning

Reply:
Ah, yes, I think you're right. We always had `listenAddress`, then added the `advertisedListenAddress`, then soft-removed the latter by having them both return the same content.